adapters/pubmatic_test.go

```diff
@@ -141,14 +202,108 @@ func TestPubmaticInvalidStatusCode(t *testing.T) {
 	}
 }
 
-func TestPubmaticMissingPublisherId(t *testing.T) {
+func TestPubmaticInvalidInputParameters(t *testing.T) {
 
-	server := httptest.NewServer(
-		http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			// Send 404
-			http.Error(w, http.StatusText(http.StatusNotFound), http.StatusNotFound)
-		}),
-	)
+	conf := *DefaultHTTPAdapterConfig
+	an := NewPubmaticAdapter(&conf, "http://localhost/test", "localhost")
+	ctx := context.TODO()
```

Comment:
Tests should use [context.Background()](https://golang.org/pkg/context/#Background) I think.

Reply:
@dbemiller  Thanks for review comments, We have updated test cases with  ctx := context.Background(). Changes from #140 are merged manually. Please let us know if you have any further queries.